src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.ts

```diff
@@ -243,4 +255,37 @@ export class TextCellComponent extends CellView implements OnInit, OnChanges {
 	protected toggleMoreActionsButton(isActiveOrHovered: boolean) {
 		this._cellToggleMoreActions.toggleVisible(!isActiveOrHovered);
 	}
+
+	public deltaDecorations(newDecorationRange: NotebookRange, oldDecorationRange: NotebookRange): void {
+		if (oldDecorationRange) {
+			this.removeDecoration(oldDecorationRange);
+		}
+
+		if (newDecorationRange) {
+			this.addDecoration(newDecorationRange);
+		}
+	}
+
+	private addDecoration(range: NotebookRange): void {
+		if (range && this.output && this.output.nativeElement) {
```

Comment:
If you think the ranges might be undefined then the parameters should be marked as optional (this will need to be fixed for strict null checking once we get around to enabling that here)